packages/webapp/src/components/VoteButton/index.tsx

```diff
@@ -30,21 +37,32 @@ const VoteButton: React.FC<{voteType: VOTE_OPTIONS}> = props => {
   const { voteType } = props;
   const voteNotSelected = (currentVote !== undefined) && currentVote !== voteType;
   const dispatch = useAppDispatch();
-  const changeVote = () => {
+
+  const changeVote = (voteType: VOTE_OPTIONS) => {
     if (currentVote || !wsConnected) return;
-    
+
     dispatch(setCurrentVote(voteType));
-    dispatch(sendVote({"nounId": nextNounId, "blockhash": blockHash, "vote": voteType}));
+    dispatch(sendVote({ "nounId": nextNounId, "blockhash": blockHash, "vote": voteType }));
   }
 
   const disabled = voteNotSelected || (!votingActive || activeAuction) || blockHash !== votingBlockHash
 
+  useEventListener("keydown", (key) => {
```

Comment:
Updated

Reply:
Yes, it could be moved up, but we'll end up with a listener with multiple `if-else` cases and a more complicated upper component since we'll need to replicate the same functions and data hooks. In the end, we'll have more Hooks, but just of a different type.